plugins/modules/ecs_taskdefinition.py

```diff
@@ -780,7 +799,8 @@ def main():
         volumes=dict(required=False, type='list', elements='dict'),
         launch_type=dict(required=False, choices=['EC2', 'FARGATE']),
         cpu=dict(),
-        memory=dict(required=False, type='str')
+        memory=dict(required=False, type='str'),
+        placement_constraints=dict(required=False, type='list', elements='dict')
```

Comment:
Given that this is a net-new argument it would be better to define the options here.

I'd also recommend adding an extra `,` on the end here: it means that when someone adds the next argument the only change is an added line rather than a changed line.

```suggestion
        placement_constraints=dict(required=False, type='list', elements='dict'),
```